readme.md

```diff
@@ -34,11 +34,26 @@ interface InjectedWindowProvider {
         /** Text to be signed */
         plaintext: string,
     ) => Promise<Signed>
+    
+    signExtrinsicWithDid: (
+        /** The extrinsic to be DID-authorized as hex-encoded string */
+        extrinsic: `0x${string}`,
+        /** KILT address that will be submitting the transaction  */
+        submitter: string,
```

Comment:
we were talking about adding an optional hint to what did is supposed to be used. Shall we add it here already, or in another update?

Reply:
We were discussing this approach with a few people and it seemed to be questionable, so I’d rather postpone this.